### PR TITLE
rolling pin for flattening dough

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
adds rolling pin to dinnerware vendor, can be used on dough to flatten it. flattened dough is probably unused so all you can do is eat it raw.
fire extinguisher drink cans and bottles can also be used to flatten dough, albeit slower than a real rolling pin

## Why / Balance
one small step for man one giant leap for construction graph pizza
can also be used as a weapon

## Technical details
no

## Media

rolling dough

pizza time

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
no

**Changelog**
:cl:
- add: Pizza recipes now require flat dough. Get a rolling pin from the dinnerware vendor.

### CODE REVIEW

**Comment by JoeHammad1844** [comment]
15 size is a little too much, 10 would make sense as it does the same damage as the crowbar

**Comment by deltanedas** [comment]
rolling pin in pocket real

**Comment by JoeHammad1844** [comment]
all is right in the world

**Comment by lzk228** [comment]
food construction real
also make all pizzas recipes requires flat dough

**Comment by deltanedas** [comment]
might as well...

**Comment by Partmedia** [comment]
Please review the [Changelog guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). As it stands, this change is probably not significant enough to warrant a Changelog.

**Comment by deltanedas** [comment]
anyone that tries to make pizza will not know why its making bread this is important unless you really want to troll chefs

**Comment by Partmedia** [comment]
> anyone that tries to make pizza will not know why its making bread this is important unless you really want to troll chefs

Earlier you had a PR description that said it changed nothing in the game. I see you have edited that now. Please remove the first "added" changelog entry because that is not significant enough, or figure out how to combine it into the second, e.g. "which are now available in the dinnerware"

File: Resources/Prototypes/Entities/Objects/Tools/tools.yml [hunk 1]
```diff
@@ -497,3 +497,27 @@
       Wood: 50
   - type: StaticPrice
     price: 25
+
+- type: entity
+  parent: BaseItem
+  id: RollingPin
+  name: rolling pin
+  description: Used to knock out the Bartender.
```
Comment (Partmedia) [inline Resources/Prototypes/Entities/Objects/Tools/tools.yml:505]: Come up with a better description for this item please.

File: Resources/Prototypes/Entities/Objects/Tools/tools.yml [hunk 2]
```diff
@@ -497,3 +497,27 @@
...
+    sprite: Objects/Tools/rolling_pin.rsi
+    state: icon
+  - type: Item
+    sprite: Objects/Tools/rolling_pin.rsi
+    size: 10
+  - type: ItemCooldown
+  - type: MeleeWeapon
+    damage:
+      types:
+        Blunt: 8
```
Comment (Partmedia) [inline Resources/Prototypes/Entities/Objects/Tools/tools.yml:517]: Damage seems excessive.

**Review by Partmedia — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Tools/tools.yml (hunk 2 above)
Comment (deltanedas) [reply Resources/Prototypes/Entities/Objects/Tools/tools.yml:517]: its the same as a crowbar which is still less than any of the chefs knives

File: Resources/Prototypes/Entities/Objects/Tools/tools.yml (hunk 1 above)
Comment (deltanedas) [reply Resources/Prototypes/Entities/Objects/Tools/tools.yml:505]: i have no creativity

File: Resources/Prototypes/Entities/Objects/Tools/tools.yml (hunk 2 above)
Comment (Partmedia) [reply Resources/Prototypes/Entities/Objects/Tools/tools.yml:517]: Please reduce it from the crowbar.

File: Resources/Prototypes/Entities/Objects/Tools/tools.yml (hunk 1 above)
Comment (Partmedia) [reply Resources/Prototypes/Entities/Objects/Tools/tools.yml:505]: "A tool used to shape and flatten dough."

File: Resources/Prototypes/Entities/Objects/Tools/tools.yml (hunk 2 above)
Comment (JoeHammad1844) [reply Resources/Prototypes/Entities/Objects/Tools/tools.yml:517]: 7 :trollface:

**Comment by moonheart08** [comment]
gas tanks should be valid rolling pins (except the plasma one which is square)

**Comment by IamVelcroboy** [comment]
> gas tanks should be valid rolling pins (except the plasma one which is square)

And baseball bats?

**Comment by Partmedia** [comment]
Thank you for your contribution.

**Pull request merged**